plugins/mediaPlayer/mediaPlayer.ts

```diff
@@ -193,6 +197,16 @@ export const initMediaPlayer = (
     },
   });
 
+  const volumeComputed = computed({
```

Comment:
id prefer exposing volume and doing:
```ts
watch(volume, (v) => {
  activeMedia.value?.setVolume(v);
});
```

Reply:
I think the current solution is more in line with what is already being done in the mediaPlayer